ghost/admin/tests/helpers/members.js

```diff
@@ -13,3 +13,9 @@ export function disableMembers(server) {
         ? server.db.settings.update({key: 'members_signup_access'}, {value: 'none'})
         : server.create('setting', {key: 'members_signup_access', value: 'none', group: 'members'});
 }
+
+export function enablePaidMembers(server) {
+    server.db.settings.find({key: 'paid_members_enabled'})
+        ? server.db.settings.update({key: 'paid_members_enabled'}, {value: true})
+        : server.create('setting', {key: 'paid_members_enabled', value: true, group: 'members'});
+}
```

Comment:
I'm not actually sure this is totally the "right" way to go about this. But it's what I found that works, at least. Alternatives I tried:

1.  adding `paid_members_enabled` directly to `mirage/fixtures/settings.js` and then loading that in with `loadFixtures` in the test file. 
2. manually creating the setting with `this.server.create('setting', ...)` in the `beforeEach` of the `beforeEach` of the `as Owner` block of tests.

Neither worked for some reason though I don't ever that could be 💯 user error.

Reply:
This is the right way to do it as it keeps it explicit 👍 The other two approaches are also valid in some cases though so it's interesting they didn't work 🤔